examples/example_feature_types.py

```diff
@@ -38,8 +37,11 @@
 # ==========================
 
 cls = autosklearn.classification.AutoSklearnClassifier(
-    time_left_for_this_task=120,
-    per_run_time_limit=30,
+    time_left_for_this_task=60,
```

Comment:
Looking at this examples, it could actually be drastically simplified - we shouldn't need the label encoder any more, and it should be sufficient to download the dataset from OpenML only once. Also, could you please add a link to the pandas example from here and vice versa as they are very related?

Reply:
Done!